src/coverlet.msbuild.tasks/CoverageResultTask.cs

```diff
@@ -54,11 +55,16 @@ public string ThresholdStat
             set { _thresholdStat = value; }
         }
 
+        public CoverageResultTask()
+        {
+            _logger = new MSBuildLogger(Log);
+        }
+
         public override bool Execute()
         {
             try
             {
-                Console.WriteLine("\nCalculating coverage result...");
```

Comment:
We should not change old output formatting `\n' here and also on other(space before text)